src/lang/german.lng

```diff
@@ -1,7 +1,7 @@
 ##grflangid 0x02
 
 STR_GRF_NAME						:{TITLE} {VERSION}
-STR_GRF_DESC						:{ORANGE}Vanilla-inspirierte Industrien{BLACK} mit neuen Industrie-Ketten, Pendlern, Industrien nahe Städten, sowie echten Einführungszeiten ab dem Jahr 1700.{}{}{SILVER}Braucht NewGRF-Fahrzeuge. Nicht kompatibel mit anderen Industrie-Sets.{}{}{ORANGE}Besuchen Sie die Website für mehr Informationen.{}{}{BLACK}Programmiert von 2TallTyler.
+STR_GRF_DESC						:{ORANGE}Vanilla-inspirierte Industrien{BLACK} mit neuen Industrie-Ketten, Pendlern, Industrien nahe Städten, sowie echten Einführungszeiten ab dem Jahre 1700.{}{}{SILVER}Braucht NewGRF-Fahrzeuge. Nicht kompatibel mit anderen Industrie-Sets.{}{}{ORANGE}Besuchen Sie die Website für mehr Informationen.{}{}{BLACK}Programmiert von 2TallTyler.
```

Comment:
Based on some research, I think it would be better to leave it as "die Jahr" instead of "die Jahre" as the former means "the year" and the latter means "the years", but only one year is explicitly mentioned. I don't have that much experience in German, so I can't say for sure.

Reply:
It's a style preference. You can write "das Jahr" in the dative with either an null-ening, but also a -e-ending. It's kind of old-fashioned, but is, imo more formal "and cool because it is so old-fashioned".
https://en.wikipedia.org/wiki/Dative_case#German
https://de.wikipedia.org/wiki/Dativ#Dativ-e
https://www.duden.de/sprachwissen/sprachratgeber/Endung-e-im-Dativ-stark-gebeugter-Maskulina-und-Neutra